docs/photoniq/event-delivery/event-delivery-filters.md

```diff
@@ -0,0 +1,165 @@
+import TabItem from '@theme/TabItem';
+
+PhotonIQ Event Delivery offers advanced filtering capabilities, allowing users to specify the exact events they want to receive. By leveraging PostgreSQL `SELECT` statement syntax, users can create filters that customize the event stream to their needs. This page outlines how to construct these queries, providing syntax guidelines, use cases, and examples.
+
+## Filters syntax
+
+Filters are defined in a JSON format, which includes fields to specify the desired event criteria. The fundamental structure of a filter includes:
+
+
+- `action`: Specifies the filter action, such as "add" to add new filters or "remove" to remove existing filters. This is only supported for WebSockets since messages cannot be sent from the client to the server in SSE. Refer to [Dynamic filter management](#dynamic-filter-management) for more details.
+
+- `initialData`: When set to `TRUE`, returns the original data after subscribing to a stream. When set to `FALSE`, it only subscribes without returning the original data. The default is `FALSE`.
+- `compress`: Indicates if messages should be sent by the server in gz-compressed form. The default is `FALSE`.
+- `once`: A boolean flag (`TRUE` or `FALSE`), indicating whether the filter should be applied just once. By defualt, it is `FALSE`. 
+- `queries`: An array of SELECT statements defining the specific events to filter.
+
+The `SELECT` syntax follows standard SQL conventions, allowing for conditions (`WHERE`), logical operators (`AND`, `OR`), and specifying particular fields or using `*` for all fields within a collection or stream.
+
+**Example JSON filter structure**
+
+```json
+{
+  "action": "add",
+  "once": "FALSE",
+  "initialData":"FALSE",
+  "queries": [
+    "select * from CollectionName where condition1",
+    "select fieldName from CollectionName where condition2 OR condition3"
+  ]
+}
+```
+
+
+## Filter use cases and examples
+
+Using event delivery queries allows for highly customizable data streams tailored to various application needs. Whether monitoring real-time updates from a specific dataset or filtering changes based on specific criteria, these queries ensure your application processes only the most relevant information.
+
+This section demonstrates how to subscribe to and filter events using practical examples. You can interact with the PhotonIQ Event Delivery Service using `wscat` for WebSockets or `curl` for SSE. These examples highlight the flexibility of event delivery queries in EDS, allowing precise control over the data stream with SQL-like syntax and dynamic filter management to meet your specific requirements.
+
+
+### Queries
+
+To subscribe to events using EDS, send a request to the [Subscribe to Stream API](https://www.macrometa.com/docs/apiEds#/paths/ws:-api-es-v1-subscribe/get). It requires the following fields:
+
+- **EDS host**: The host where the EDS service is running.
+- **x-customer-id**: The is used to authenticate the request.
```

Comment:
Authenticate may not be right word here I think. Probably validate could be used. For authentication we have an apiKey that is provided like this example - 
wscat -c 'wss://nbapoc-us-east.photoniq.macrometa.io/api/es/v1/subscribe?x-customer-id=nba_admin&fabric=courtside&apiKey=ptq_MV6VuirCMDz_Kyejc8NLmVFKeVx8XxhYWw3mCVKMP3v2p&type=collection&filters={"action": "add", "once": "TRUE", "initialData":"TRUE", "queries": ["select * FROM game_live where _key='\''0012300001'\''", "select * FROM game_live where _key='\''0012300002'\''"]}' | sed -r "s/\x1B\[([0-9]{1,2}(;[0-9]{1,2})?)?[mGK]//g"

This API key is to be generated using the Photoniq Auth service procedure.